lib/system.g

```diff
@@ -92,7 +92,7 @@ BIND_GLOBAL( "GAPInfo", rec(
       rec( short:= "B", default := "",    arg := "<name>", help := [ "current architecture"] ),
       rec( short:= "D", default := false, help := ["enable/disable debugging the loading of files"] ),
       rec( short:= "M", default := false, help := ["disable/enable loading of compiled modules"] ),
-      rec( short:= "N", default := false, help := ["unused, for backward compatibility only"] ),
+      rec( short:= "N", default := false, help := ["do not use hidden implications"] ),
```

Comment:
@frankluebeck The comment above says "These options must be kept in sync with those in system.c" but that file is not even modified by this PR. Am I missing something?